utils.py

```diff
@@ -451,7 +430,7 @@ def process_data_request(render_me, path, subj_text):
         subj_text = f"{subj_text} Data for {terms[0]} through {terms[-1]}"
     else:
         # If there is only one term, just show that term
-        subj_text = f"{subj_text} Data for {terms[0]}"
+        subj_text = f"{subj_text} \n Data for {terms[0]}"
```

Comment:
I am not sure why you did this, but let's leave it as a single line:
```suggestion
subj_text = f"{subj_text} Data for {terms[0]}"
```